src/main/java/flaxbeard/steamcraft/block/BlockSteamCharger.java

```diff
@@ -22,8 +17,16 @@
 import net.minecraft.world.IBlockAccess;
 import net.minecraft.world.World;
 import net.minecraftforge.common.util.ForgeDirection;
-
-import java.util.Random;
+import tconstruct.library.tools.ToolCore;
```

Comment:
Was this a build *without* TConstruct installed? I.e., just Flaxbeard's Steam Power and Forge?